alarm/ExceptionManager.js

```diff
@@ -164,6 +177,9 @@ module.exports = class {
           callback(err);
         });
       });
+
+      Bone.submitException('allow', exception);
```

Comment:
Shall we add exception to cloud only after exception is added to local redis successfully?

Reply:
no need, this only captures user's intention, no matter it is locally stored successfully or not.